view/frontend/templates/cards.phtml

```diff
@@ -0,0 +1,57 @@
+<script type="text/javascript">// <![CDATA[
+require([
+        'jquery',
+        'Magento_Ui/js/modal/confirm'
+    ],
+    function($, confirmation) {
+         $('#cc-delete').on('click', function(event){
+             event.preventDefault();
+                 confirmation({
+             title: 'Delete Card',
+             content: 'Are you sure?',
+             actions: {
+                 confirm: function(){
+                    window.location.href = event.currentTarget.href;
+                 },
+                 cancel: function(){
+                   return false;
+                 },
+                 always: function(){}
+             }
+           });
+       });
+      }
+);
+// ]]></script>
+
+<?php
+$cards = $block->getCards();
+?>
+
+<?php if (count($cards['data']) !== 0): ?>
+    <div class="table-wrapper my-credit-cards">
+        <table class="data table table-credit-cards" id="my-orders-table">
+            <caption class="table-caption"><?= /* @noEscape */ __('Stored Payment Methods') ?></caption>
+            <thead>
+                <tr>
+                    <th scope="col" class="col card-number"><?= /* @noEscape */ __('Card Number') ?></th>
+                    <th scope="col" class="col expire"><?= /* @noEscape */ __('Expiration Date') ?></th>
+                    <th scope="col" class="col card-type"><?= /* @noEscape */ __('Type') ?></th>
+                    <th scope="col" class="col actions">&nbsp;</th>
+                </tr>
+            </thead>
+            <tbody>            
+<?php foreach($cards['data'] as $card): ?>
+                <tr>
+                    <td><?php echo ' **** **** **** ' . $card['last_digits']; ?></td>
+                    <td><?php echo $card['expiration_month']; ?>/<?php echo $card['expiration_year']; ?></td>
+                    <td><?php echo $card['brand'] ?></td>
+                    <td><a id="cc-delete" href="<?php echo $block->getDeleteLink($card)?>"><?= /* @noEscape */ __('Delete') ?></a></td>
```

Comment:
@jacstn id is referencing to only a specific one unique element. In this case, should be `class`.